port.go

```diff
@@ -305,7 +311,7 @@ func (svc *PortServiceOp) ModifyPort(ctx context.Context, req *ModifyPortRequest
 		ProductType:           PRODUCT_MEGAPORT,
 		Name:                  req.Name,
 		CostCentre:            req.CostCentre,
-		MarketplaceVisibility: req.MarketplaceVisibility,
+		MarketplaceVisibility: &req.MarketplaceVisibility,
```

Comment:
Same thing as the MCR, if the user doesn't want to specify it it should also be a pointer in the `ModifyPortRequest`